helpers/calendar.py

```diff
@@ -0,0 +1,14 @@
+from helpers.database import db
+
+
+def update_calendar(calendar, calendar_key, room):
+        if not room['firebaseToken']:
+            room['firebaseToken'] = ''
+        if not calendar:
+            key = len(db.keys('*')) + 1
+            db.hmset(key, {'calendar_id': room['calendarId'], 'firebase_token': room['firebaseToken']})
+            return None
```

Comment:
Why are we returning None on these functions?

Reply:
Thanks for pointing it out, the return has been removed.